### ISSUE TITLE
[zandronum] Animation bug - toxin walls (E2M3)

### ISSUE THREAD

**Issue by Coincident** [KuriKai/DHTP#8]
Hello.
While playing brutal doom (on zandronum) I've found a bug of an animated wall on E2M3. It's an animation of pipes that are leaking green toxins onto the damaging floors below.

It can be seen on this youtube video that someone else has made before:
https://www.youtube.com/watch?v=iFQKRXTX8l0#t=740 (at 12:20)

**Comment by KuriKai** [comment]
Looks like it is loading textures from the TNT folder. Could be an engine problem. Does this problem still exist?

**Comment by Coincident** [comment]
Yes. Eventually I changed from zandronum to gzdoom, and it also happens there.

**Comment by goldenwarriorgw** [comment]
https://youtu.be/B1TrMuhklkI I think he's referring to this

**Comment by KuriKai** [comment]
Does this still happen on the latest pack?

**Comment by goldenwarriorgw** [comment]
@KuriKai Nope

**Comment by goldenwarriorgw** [comment]
On Latest GZDoom-No 
Zandronum-It's complicated
